components/brave_wallet/browser/eth_data_builder.cc

```diff
@@ -23,6 +30,32 @@ bool IsValidHostLabelCharacter(char c, bool is_first_char) {
          (c >= '0' && c <= '9') || (!is_first_char && c == '-') || c == '_';
 }
 
+absl::optional<std::string> ChainIdToVersion(const std::string& symbol,
```

Comment:
Should this be inside the `unstoppable_domains` namespace?

Reply:
That's in anonymous namespace, only for local use